File: Basic Programming/IO/Goki and his breakup.py
#https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/tds-and-his-breakup/

t = int(input())
m = int(input())

for i in range(t):
    a = int(input())
    if a < m:
        print("NO")
    else:
        print("YES")

File: Basic Programming/IO/Goki and his breakup.cpp
//https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/tds-and-his-breakup/


#include<iostream>

using namespace std;

int main()
    {
        int t,m,a;
        cin>>t>>m;
        while(t--) 
            {
                cin>>a;
                if(a<m)
                    cout<<"NO";
                else
                    cout<<"YES";
                cout<<endl;
            }
        
        return 0;
    }